### PR TITLE
:sparkles: Implement Top Section of AboutScreen

### LINKED ISSUE #429: Implement Top Section of AboutScreen
**Idea Description**
Implement the top section of the AboutScreen, as divided into three parts in the design on [Figma](https://www.figma.com/file/MbElhCEnjqnuodmvwabh9K/DroidKaigi-2023-App-UI). The task includes:
   - Implementing the design for the top section only.
   - Adding the design to the existing empty implementation in AboutDroidKaigiDetail.
   - No need to create any navigation behavior; simply replicating the design is sufficient.

**Reference images and links**
image

### PR DESCRIPTION
**Pull request by Corvus400**
## Issue
- close #429 

## Overview (Required)
- The top section of the About screen has been implemented.
- As mentioned in the TODO, I think it would be possible to make content containing URLs clickable on the session screen.
- Once the session screen is mostly implemented, we would like to be able to use the functions implemented in this screen as common components.

## Where I need help .
- I would like to be able to preview in English and Japanese, but it only previews in Japanese.
- Am I using StringsBindings incorrectly? 🤔

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

**Comment by takahirom** [comment]
Thanks!
Maybe we have to scroll the about screen for tests 👀 
You can refer to the README
https://github.com/DroidKaigi/conference-app-2023#robot-section-how-to-test

File: core/testing/src/main/java/io/github/droidkaigi/confsched2023/testing/robot/AboutScreenRobot.kt [hunk 1]
```diff
@@ -55,4 +60,15 @@ class AboutScreenRobot @Inject constructor(
         composeTestRule.waitForIdle()
         testDispatcher.scheduler.advanceUntilIdle()
     }
+
+    fun scrollAboutScreen() {
```
Comment (takahirom) [inline core/testing/src/main/java/io/github/droidkaigi/confsched2023/testing/robot/AboutScreenRobot.kt:64]: You can provide the scrolled screenshot test to check the scrolled screen 😉  You don't have to do it in this PR though.

File: feature/about/src/main/java/io/github/droidkaigi/confsched2023/about/AboutStrings.kt [hunk 2]
```diff
@@ -5,13 +5,35 @@ import io.github.droidkaigi.confsched2023.designsystem.strings.Strings
 import io.github.droidkaigi.confsched2023.designsystem.strings.StringsBindings
 
 sealed class AboutStrings : Strings<AboutStrings>(Bindings) {
```
Comment (takahirom) [inline feature/about/src/main/java/io/github/droidkaigi/confsched2023/about/AboutStrings.kt:7]: Great!

File: feature/about/src/main/java/io/github/droidkaigi/confsched2023/about/component/AboutDroidKaigiDetail.kt [hunk 3]
```diff
@@ -1,10 +1,93 @@
...
+    }
+}
+
+@Preview(locale = "en", uiMode = Configuration.UI_MODE_NIGHT_NO)
+@Preview(locale = "ja", uiMode = Configuration.UI_MODE_NIGHT_NO)
+@Composable
+fun AboutDroidKaigiDetailPreview() {
+    KaigiTheme {
+        Surface {
+            AboutDroidKaigiDetail()
```
Comment (takahirom) [inline feature/about/src/main/java/io/github/droidkaigi/confsched2023/about/component/AboutDroidKaigiDetail.kt:79]: 👍

File: feature/about/src/main/java/io/github/droidkaigi/confsched2023/about/AboutStrings.kt [hunk 4]
```diff
@@ -5,13 +5,35 @@ import io.github.droidkaigi.confsched2023.designsystem.strings.Strings
...
+                PlaceTitle -> "場所"
+                PlaceDescription -> "ベルサール渋谷ガーデン"
+                PlaceLink -> "地図を見る"
+            }
         },
         Lang.English to { item, bindings ->
-            TODO()
+            when (item) {
+                Title -> "About"
+                Description -> "DroidKaigi is an engineer-driven Android conference held to share and communicate Android technical information."
```
Comment (takahirom) [inline feature/about/src/main/java/io/github/droidkaigi/confsched2023/about/AboutStrings.kt:30]: This is from the website https://2023.droidkaigi.jp/en/ 🌐 

```suggestion
                Description -> "DroidKaigi is a conference tailored for Android developers."
```

File: feature/about/src/main/java/io/github/droidkaigi/confsched2023/about/component/AboutDroidKaigiDetailSummaryCardRow.kt [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+        Spacer(modifier = Modifier.width(12.dp))
+        ClickableLinkText(
+            style = MaterialTheme.typography.bodyMedium,
+            content = content,
+            onLinkClick = {}, // TODO Go to FloorMapScreen
+        )
+    }
+}
+
+// TODO The following components can be commonized and made available on the session screen.
```
Comment (takahirom) [inline feature/about/src/main/java/io/github/droidkaigi/confsched2023/about/component/AboutDroidKaigiDetailSummaryCardRow.kt:57]: 👍

**Review by takahirom — APPROVED** [review]
LGTM! Please take a look at the text for English 🙏

**Comment by Corvus400** [comment]
@takahirom 
Thank you for your review!
I'll fix the parts that need to be fixed and commit it!
By the way, I want to display Japanese and English in @Preview but it doesn't seem to work.
Am I using my StringsBindings incorrectly here?

**Comment by takahirom** [comment]
I'm not sure why it doesn't work. 👀

**Comment by Corvus400** [comment]
@takahirom 
We have completed the correction [here](https://github.com/DroidKaigi/conference-app-2023/pull/481/commits/ff98149f690d73396316df2dd1d3cb539fbdf205)!
Please check again. 🙏

**Comment by takahirom** [comment]
Good job!

**Comment by tkhs0604** [comment]
@Corvus400 
Thank you for your contribution! Looks excellent!

> By the way, I want to display Japanese and English in Preview annotation but it doesn't seem to work.

I created a new issue for the above point: https://github.com/DroidKaigi/conference-app-2023/issues/509
If interested, it would be appreciated if you could challenge to resolve it 😉

**Pull request merged**